instance/tests/test_tasks.py

```diff
@@ -59,13 +63,14 @@ def test_watch_pr_new(self, mock_get_username_list, mock_get_pr_list_from_userna
         New PR created on the watched repo
         """
         mock_get_username_list.return_value = ['itsjeyd']
-        pr = github.PR(
+        settings = 'WATCH: true\r\nDATABASE_URL: mysql://db.opencraft.com\r\nMONGO_URL: mongo://mongo.opencraft.com\r\n'
```

Comment:
To avoid putting too much into this test, can you keep it to the "fake" `WATCH: true` content, and create separate tests for additional logic?